batavia/modules/time.js

```diff
@@ -1,7 +1,9 @@
 
 batavia.modules.time = {
+    _startTime: new Date().getTime(),
     clock: function() {
-        return new Date().getTime();
+        // TODO: this does something a little different in CPython.
```

Comment:
Can you elaborate on what "a little different" means here? I know this is going to be one of those functions where some discrepancy from CPython is to be expected, but it would be good to document exactly what that entails.